lib/transforms/workflow.js

```diff
@@ -0,0 +1,62 @@
+const parseSmartCommit = require('./smart-commit');
+
+function mapStatus({ status, conclusion }) {
+  if (status === 'queued') {
+    return 'in_progress';
+  } else if (status === 'in_progress') {
+    return 'in_progress';
+  } else if (status === 'completed') {
+    if (conclusion === 'success' || conclusion === 'neutral') {
+      return 'successful';
+    } else if (conclusion === 'failure' || conclusion === 'timed_out') {
+      return 'failed';
+    } else if (conclusion === 'cancelled' || conclusion === 'stale') {
+      return 'cancelled';
+    } else if (conclusion === 'action_required') {
+      return 'pending';
+    } else {
+      return 'unknown';
+    }
+  } else {
+    return 'unknown';
+  }
```

Comment:
these status mappings are getting unruly...

Reply:
Just added some documentation and changed it to a switch.